package.json

```diff
@@ -2,18 +2,7 @@
   "name": "publish-browser-extension",
   "description": "Easily publish web extensions to their stores",
   "version": "3.0.2",
-  "engines": {
-    "node": "^18.0.0 || >=20.0.0"
-  },
```

Comment:
Why was this one deleted? This is for consumers and is still very valuable metadata.

Reply:
https://github.com/aklinker1/publish-browser-extension/commit/735e15263b32eede2fea37ae95bafee4f4610a0e